helm/charts/kratos/README.md

```diff
@@ -100,15 +100,16 @@ A ORY Kratos Helm chart for Kubernetes
 | serviceMonitor.scrapeInterval | string | `"60s"` | Interval at which metrics should be scraped |
 | serviceMonitor.scrapeTimeout | string | `"30s"` | Timeout after which the scrape is ended |
 | serviceMonitor.tlsConfig | object | `{}` | TLS configuration to use when scraping the endpoint |
-| statefulSet.annotations | object | `{}` |  |
-| statefulSet.extraContainers | object | `{}` | If you want to add extra sidecar containers. |
-| statefulSet.extraEnv | list | `[]` |  |
-| statefulSet.extraInitContainers | object | `{}` | If you want to add extra init containers. These are processed before the migration init container. |
-| statefulSet.extraVolumeMounts | list | `[]` |  |
-| statefulSet.extraVolumes | list | `[]` | If you want to mount external volume For example, mount a secret containing Certificate root CA to verify database TLS connection. |
-| statefulSet.labels | object | `{}` |  |
-| statefulSet.nodeSelector | object | `{}` | Node labels for pod assignment. |
-| statefulSet.resources | object | `{}` |  |
+| statefulset.annotations | object | `{}` |  |
```

Comment:
The docs are auto-generated, no need to modify it :)

Reply:
I see. I think my changes can just be overwritten with the generated docs right?